src/redisearch_api.h

```diff
@@ -183,6 +183,8 @@ MODULE_API_FUNC(int, RediSearch_QueryNodeGetFieldMask)(RSQNode* qn);
 
 MODULE_API_FUNC(RSResultsIterator*, RediSearch_GetResultsIterator)(RSQNode* qn, RSIndex* sp);
 
+MODULE_API_FUNC(void, RediSearch_SetMaxResultsToUseCT)(size_t num);
```

Comment:
This doesn't need to be an API function; just access RSGlobalConfig directly from the tests.

Reply:
This should defensively be an api function cause the user might want to change this value for performance running.

Reply:
Hrm, in that case, let's rename it to something like SetCriteriaTesterThreshold

Reply:
Done